examples/ec2/snapshotcreatevolumepermission.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: SnapshotCreateVolumePermission
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/snapshotcreatevolumepermission
+    upjet.upbound.io/manual-intervention: "Problem on deletion. EBSSnapshot deleting beforeSnapshotCreateVolumePermission, what leads to stuck deletion proccess"
```

Comment:
@dverveiko I am wondering if we could avoid this manual intervention using `post-assert` [hooks](https://github.com/upbound/uptest#hooks)? You can check [this](https://github.com/upbound/platform-ref-gcp/blob/6208fdfcd070b1ff01ba2ff2825a12f12852aa6d/examples/cluster-claim.yaml#L7) as an example.